.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "restructuredtext.preview.sphinx.disabled": true,
+    "restructuredtext.linter.disabled": true
+}
```

Comment:
Please, remove this file from your pull request. You can use a [global .gitignore](https://stackoverflow.com/a/7335487/939364) to have Git ignore such files in your system.